src/Containers/Profile/Profile.scss

```diff
@@ -0,0 +1,194 @@
+@import '../../assets/sass/global.scss';
+#profile {
+  input,
+  button {
+    width: 80%;
+  }
+  button {
+    width: 80%;
+  }
+  label {
+    color: #444;
+  }
+  textarea {
+    border: 2px solid #efefef;
+    margin-bottom: 10px;
+    padding: 0 6px;
+    width: 90%;
+    background: #fafafa;
+    &:focus {
+      outline: none !important;
+      border: 2px solid rgba(#aca9a9, 0.5);
+      box-shadow: 2px 0 6px rgba(#efefef, 0.8);
+    }
+  }
+  .navbar {
+    background-color: white;
+  }
+}
+
+.body {
+  padding-left: 14%;
+
+  div,
+  .profile {
+    background-color: #f5f5f5;
+  }
+  #pic {
+    img {
+      width: 70%;
+    }
+    .uname {
+      padding: 50px 10;
+    }
+  }
+}
+@media only screen and (max-width: 700px) {
+  #profile {
+    padding-left: 10%;
+    #pic {
+      display: flex;
+      height: 100px;
+      .form-box {
+        width: 40%;
+        height: 0;
+        padding-top: 0%;
+        img {
+          width: 110%;
+        }
+      }
+
+      .uname {
+        padding: 0;
+      }
+    }
+  }
+  input,
```

Comment:
same here it'll be better if you just move it inside of `#profile`, I think you can even do the same to the textarea bellow